testcases/crypto/ec_func.c

```diff
@@ -725,11 +1309,16 @@ run_TransferECCKeyPairSignVerify()
 	}
 
 	goto done;
+
 testcase_cleanup:
-	funcs->C_DestroyObject(session, publ_key);
-	funcs->C_DestroyObject(session, priv_key);
-	funcs->C_DestroyObject(session, secret_key);
-	funcs->C_DestroyObject(session, unwrapped_key);
+    if (publ_key != CK_INVALID_HANDLE)
+        funcs->C_DestroyObject(session, publ_key);
+    if (priv_key != CK_INVALID_HANDLE)
+        funcs->C_DestroyObject(session, priv_key);
+    if (secret_key != CK_INVALID_HANDLE)
+        funcs->C_DestroyObject(session, secret_key);
+    if (unwrapped_key != CK_INVALID_HANDLE)
```

Comment:
Can you also align the if below just like this one?
During build, compiler is complaining about misalignment on this code.